documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -6,26 +6,53 @@
 = Handling high volumes of messages
 
 [role="_abstract"]
-If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it.
+If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it. The configuration can help manage throughput and issues with latency. 
 
-The flush pipeline for data replication is *source topic -> (Kafka Connect) source message queue -> producer buffer -> target topic*.
-An offset flush timeout period (`offset.flush.timeout.ms`) is the time to wait for the producer buffer (`producer.buffer.memory`) to flush and offset data to be committed.
-Try to avoid a situation where a large producer buffer and an insufficient offset flush timeout period causes a _failed to flush_ or _failed to commit offsets_ type of error.
+MirrorMaker 2.0 uses an internal producer client (`KafkaProducer`) to replicate data from the source to target Kafka cluster. 
+To control the streaming of high volumes of data, you can tune the following producer configuration for optimal throughput: 
 
-This type of error means that there are too many messages in the producer buffer, so they can't all be flushed before the offset flush timeout is reached.
+* Producer batch size 
+* Producer buffer size 
```

Comment:
I would remove `buffer.size`, as it is a sub-optimal solution.
You may fix the problem by reducing this value, but the overall throughput would be bad.
Anyway, if we decide to keep, then we should explain how to set it.

Reply:
Actually meaning the producer buffer memory here, which we describe later. I've changed to that